src/utils/price-utils.ts

```diff
@@ -58,22 +63,6 @@ const getUsdVolume = async (token: TokenInfo, amount: BigNumber): Promise<number
     return +adjustedAmount1 * price / BASIS_POINTS_DIVISOR;
 }
 
-export const isPriceImpactOverThreshold = async (
-    tokenIn: TokenInfo, 
-    tokenOut: TokenInfo, 
-    amountIn: BigNumber, 
-    amountOut: BigNumber,
-    threshold = 0.05 // 5%
-) => {
-    const [volumeIn, volumeOut] = await Promise.all([
-        getUsdVolume(tokenIn, amountIn),
-        getUsdVolume(tokenOut, amountOut)
-    ]);
-
-    // if no volume able to be fetched, assume no error
-    if (!volumeIn || !volumeOut) return false;
-
-    // 1 - (usdVolume2 / usdVolume1) is the formula for the fractional 
-    // difference between the USD value in and out
-    return 1 - (volumeOut / volumeIn) > threshold;
-}
+export const getPriceImpact = (usdVolumeIn: number, usdVolumeOut: number) => {
+    return 1 - (usdVolumeOut / usdVolumeIn);
+}
```

Comment:
Let's use implicit return when possible
```suggestion
export const getPriceImpact = (usdVolumeIn: number, usdVolumeOut: number) =>
    1 - (usdVolumeOut / usdVolumeIn);
```

Reply:
done